packages/pyiceberg/test_pyiceberg.py

```diff
@@ -0,0 +1,41 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(
+    packages=[
+        "pyiceberg",
```

Comment:
You can just put `pyiceberg` here, and remove other packages.

Reply:
Hey @ryanking13 / @rth / @hoodmane, I think I'm hitting the same issue here as this issue with these pure python packages:

```
  'click',
  'fsspec',
  'rich',
  'sortedcontainers', 
  'strictyaml'
```
I am a bit confused by the outcome or next steps and I don't see anything exactly answering this question in the [FAQ](https://pyodide.org/en/stable/usage/faq.html).

https://github.com/pyodide/pyodide/issues/1791#issuecomment-898295481 

Thanks :) I'll push what I have shortly.

Reply:
You need to put all those pure packages to `requirements/run`, then pyodide will load all those dependencies when loading `pyiceberg`.

Make sure you added `meta.yaml` for all those pure python packages as well. For now, we require you to add meta.yaml for pure python packages as well as non-pure python packages to make lockfile (There is a related discussion here: https://github.com/pyodide/pyodide/issues/2580).